README.md

```diff
@@ -37,6 +37,16 @@ This role requires:
 - [Ubuntu - 14.04 (Trusty Tahr)](http://releases.ubuntu.com/14.04/)
 - [Ubuntu - 16.04 (Xenial Xerus)](http://releases.ubuntu.com/16.04/)
 
+### In Testing
+
+- Windows 2012
+- Windows 2016
+
+### Future Releases
+
+- OpenBSD
```

Comment:
We actually already support `OpenBSD` and so far no one has asked for NetBSD support. 

For this and for the immediate future (until Appveryor or something similar is setup), can we move the Windows 2012/2016 under the ` #### Supported manually (compatibility not always guaranteed)`  section? Then, once I have some automatic testing setup for Windows, we'll pull them back out under the automatically tested section above.

Reply:
My fork is out of date, and i failed to re-base your changes before pushing these. I guess the future releases was part of the old readme. I've updated the Windows versions with wikipedia links (can't find official MS pages for those on google)